src/bundle/ui-dev/src/modules/common/popup/popup.component.js

```diff
@@ -58,6 +59,7 @@ const Popup = ({
     const modalClasses = createCssClassNames({
         'c-popup modal fade': true,
         'c-popup--no-header': noHeader,
+        'c-popup--has-subheader': !noHeader && subheader,
```

Comment:
is there no such scenario that modal will have both header and subheader? (also, it's strange to have subheader and no header, as theoretically without header subheader becomes header :D )

Reply:
Currently there is no such scenario